lib/internal/bootstrap_node.js

```diff
@@ -268,15 +268,29 @@
     var console;
     Object.defineProperty(global, 'console', {
       configurable: true,
-      enumerable: true,
+      enumerable: false,
       get: function() {
         if (!console) {
           console = NativeModule.require('console');
           installInspectorConsoleIfNeeded(console,
                                           inspectorConsole,
                                           wrapConsoleCall);
         }
+        Object.defineProperty(global, 'console', {
+          configurable: true,
+          writable: true,
+          enumerable: false,
+          value: console
+        });
```

Comment:
@addaleax I tried that. global.console = console fails at test/common.js:407 with message 
```
AssertionError: Unexpected global(s) found: console
```